Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM qazbnm456/fbctf_base:latest
```

Comment:
Ah, I see. I was just curious since the documentation states only Ubuntu 14.04 is needed.